IDESettings/.clang-format

```diff
@@ -0,0 +1,116 @@
+---
+Language: Cpp
+AccessModifierOffset: -4
+AlignAfterOpenBracket: DontAlign
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlines: Left
+AlignEscapedNewlinesLeft: true
+AlignOperands: false
+AlignTrailingComments: false
```

Comment:
true?

Reply:
dunno, we can certainly do that it's fancy :)

Reply:
changed